examples/mixture/plot_gmm_init.py

```diff
@@ -0,0 +1,176 @@
+"""
+==========================
+GMM Initialization Methods
+==========================
+
+Examples of the different methods of initialization in Gaussian Mixture Models
+
+See :ref:`gmm` for more information on the estimator.
+
+Here we generate some sample data with four easy to identify clusters.The
+purpose of this example is to show the four different methods for the
+initialization parameter *init_param*.
+
+- *kmeans* - performs kmeans clustering on the dataset to produce the initial
```

Comment:
I'm not so happy with this being duplicated

Reply:
I'm not entirely sure what you mean, is it that it describes the initialization choices in both the plot and the documentation?

Reply:
Yes, it just tends to be hard to keep detailed descriptions in sync.

Reply:
Though I can see why you've done it.